bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestAuth.java

```diff
@@ -48,7 +66,7 @@ public void testBatchJwt() throws IOException {
 
     // Prevent the test from hanging if auth fails
     config.set("google.bigtable.rpc.use.timeouts", "true");
-    config.set("google.bigtable.rpc.timeout.ms", "10000");
+    config.set("google.bigtable.rpc.timeout.ms", "20000");
```

Comment:
Why did this need to increase?

Reply:
Not sure why but it kept failing on with ~5-6 extra seconds. 
It might be because low bandwidth(~50Mbps) or could be low connection boot time(bigtable regions were all `us-central1-f`).

Reply:
can you try changing the connection pool size to 1 and see if that solves the problem?

Reply:
Thanks for the suggestion, It works fine with the connection pool size of 1.
Reverted back the timeout and added connection pool config.